### PR TITLE
Use single quote

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | /
| Type?         | improvement
| Category?     | CO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | /
| How to test?  | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11740)

### CODE REVIEW

File: admin-dev/filemanager/include/utils.php
Comment (jolelievre) [inline admin-dev/filemanager/include/utils.php]: probably a conflict here

**Review by jolelievre — CHANGES_REQUESTED** [review]

File: admin-dev/filemanager/include/utils.php
Comment (jolelievre) [reply admin-dev/filemanager/include/utils.php]: and other conflicts to fix from merged PRs
the rest is ok for me

File: var/SymfonyRequirements.php
Comment (Quetzacoalt91) [inline var/SymfonyRequirements.php]: Note this file may loose these changes at the first composer update.

**Review by Quetzacoalt91 — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
Conflicts must be resolved before going further in the review process. :)

**Comment by MathiasReker** [comment]
@Quetzacoalt91 @jolelievre Conflicts solved.

File: var/SymfonyRequirements.php
Comment (MathiasReker) [reply var/SymfonyRequirements.php]: Yes, I understand.

File: admin-dev/filemanager/execute.php
Comment (jolelievre) [inline admin-dev/filemanager/execute.php:37]: why change the original code? this str_replace is is used for legitimate safety reasons (null byte injection)
we recently fixed a few security breaches in this component, so I prefer not to modify its behavior if not necessary

File: admin-dev/filemanager/execute.php
Comment (jolelievre) [inline admin-dev/filemanager/execute.php]: same thing here

**Review by jolelievre — CHANGES_REQUESTED** [review]
do not remove the null byte injection protection

**Comment by MathiasReker** [comment]
Sorry, I will be more careful next time I solve conflicts.

File: admin-dev/filemanager/execute.php
Comment (PierreRambaud) [inline admin-dev/filemanager/execute.php]: You missed something here, this one should not be commit :) same below.

File: var/SymfonyRequirements.php
Comment (PierreRambaud) [reply var/SymfonyRequirements.php]: You must revert changes here :/

Comment (MathiasReker) [reply var/SymfonyRequirements.php]: Okay, I will.

Comment (MathiasReker) [reply var/SymfonyRequirements.php]: Fixed

File: admin-dev/filemanager/execute.php
Comment (MathiasReker) [reply admin-dev/filemanager/execute.php]: Thanks! Sorry, I messed a little up. Nice you catches my mistakes! :)

**Comment by MathiasReker** [comment]
All should be good now.

File: tools/parser_sql/utils/PHPSQLParserUtils.php
Comment (jolelievre) [inline tools/parser_sql/utils/PHPSQLParserUtils.php]: `# -> //` fixed in https://github.com/PrestaShop/PrestaShop/pull/11699

File: admin-dev/filemanager/include/php_image_magician.php
Comment (jolelievre) [inline admin-dev/filemanager/include/php_image_magician.php]: Some `#` left, I thought this was fixed in another PR that hase been merged https://github.com/PrestaShop/PrestaShop/pull/11699
problem during rebase maybe?

File: tools/parser_sql/processors/AbstractProcessor.php
Comment (jolelievre) [inline tools/parser_sql/processors/AbstractProcessor.php]: `# -> //`  fixed in https://github.com/PrestaShop/PrestaShop/pull/11699

File: tools/parser_sql/processors/CreateDefinitionProcessor.php
Comment (jolelievre) [inline tools/parser_sql/processors/CreateDefinitionProcessor.php]: `# -> //` fixed in https://github.com/PrestaShop/PrestaShop/pull/11699

File: tools/parser_sql/processors/CreateDefinitionProcessor.php
Comment (jolelievre) [inline tools/parser_sql/processors/CreateDefinitionProcessor.php]: `# -> //` fixed in https://github.com/PrestaShop/PrestaShop/pull/11699

File: tools/parser_sql/processors/SQLProcessor.php
Comment (jolelievre) [inline tools/parser_sql/processors/SQLProcessor.php]: `# -> //` fixed in https://github.com/PrestaShop/PrestaShop/pull/11699

File: tools/parser_sql/processors/SQLProcessor.php
Comment (jolelievre) [inline tools/parser_sql/processors/SQLProcessor.php]: `# -> //` fixed in https://github.com/PrestaShop/PrestaShop/pull/11699

**Review by jolelievre — CHANGES_REQUESTED** [review]
Careful this PR introduces reverts for `# -> //` fixed in https://github.com/PrestaShop/PrestaShop/pull/11699
Maybe you didn't rebase develop? and no conflicts appeared

File: tools/parser_sql/processors/SQLProcessor.php
Comment (MathiasReker) [reply tools/parser_sql/processors/SQLProcessor.php]: Fixed, but many of those comments should not be short comments. Thats why I would make an other PR for them :) But lets have them in this way for now. See latest commit.

**Comment by jolelievre** [comment]
Waow I didn't expect so many changes!! :P
You're some of these comments should use the block syntax, do you think it will be fixed in https://github.com/PrestaShop/PrestaShop/pull/11747 ? Or in another PR?

**Review by jolelievre — DISMISSED** [review]
Ok for me, waiting for a second approval from @PrestaShop/prestashop-core-developers

**Comment by MathiasReker** [comment]
> 
> 
> Waow I didn't expect so many changes!! :P
> You're some of these comments should use the block syntax, do you think it will be fixed in #11747 ? Or in another PR?

Exactly many of them need block syntax. I will fix it in an other PR. I would like to have  #11747 merged asap.

**Comment by MathiasReker** [comment]
Fixed conflicts

**Comment by MathiasReker** [comment]
Travis should pass now =)

**Comment by MathiasReker** [comment]
I don't understand why Travis CI does not pass?

**Comment by jolelievre** [comment]
@MathiasReker I reloaded it, seems like it's good!

File: tools/parser_sql/lexer/LexerSplitter.php
Comment (jolelievre) [inline tools/parser_sql/lexer/LexerSplitter.php]: not sure about this one..

**Review by jolelievre — CHANGES_REQUESTED** [review]

File: classes/helper/HelperForm.php
Comment (jolelievre) [inline classes/helper/HelperForm.php:29]: this alignment doesn't seem good.. did it come from php-cs-fixer?

**Review by jolelievre — CHANGES_REQUESTED** [review]

File: classes/helper/HelperForm.php
Comment (MathiasReker) [reply classes/helper/HelperForm.php:29]: Yes, it did. I agree with you! But it solved the lint issue reported by travis.

File: tools/parser_sql/lexer/LexerSplitter.php
Comment (MathiasReker) [reply tools/parser_sql/lexer/LexerSplitter.php]: yes, it looks weird. I will revert it.

Comment (MathiasReker) [reply tools/parser_sql/lexer/LexerSplitter.php]: Should be: `',', '@', ' ', '+', '-', '*', '/', ';',);`

Comment (MathiasReker) [reply tools/parser_sql/lexer/LexerSplitter.php]: and `);` on next line

Comment (MathiasReker) [reply tools/parser_sql/lexer/LexerSplitter.php]: Fixed

**Review by jolelievre — APPROVED** [review]
Requires another approval from @PrestaShop/prestashop-core-developers

File: classes/helper/HelperForm.php
Comment (MathiasReker) [reply classes/helper/HelperForm.php:29]: I fixed it.

**Comment by matks** [comment]
Dear @MathiasReker, as discussed together and following our [Contributing rules](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requests-requirements) I must unfortunately close this PR as it is too large to be thoroughly reviewed. Thank you for the contribution.

**Pull request closed without merging**
